pool/src/transaction_pool.rs

```diff
@@ -215,10 +215,9 @@ impl TransactionPool {
 
 	/// Whether the transaction is acceptable to the pool, given both how
 	/// full the pool is and the transaction weight.
-	fn is_acceptable(&self, tx: &Transaction) -> Result<(), PoolError> {
+	fn is_acceptable(&mut self, tx: &Transaction) -> Result<(), PoolError> {
```

Comment:
If we're going to evict existing txs from the pool we should rework this and not do it in `is_acceptable()`.

Reply:
Fixed.